mitodl/mitxpro
# fix redirect url after signin

#### What are the relevant tickets?
closes #542 

#### What's this PR do?
Fix redirect url.

#### How should this be manually tested?

- Logout
- Hit `/signing/checkout`
- Sign In
- You should redirect to checkout page instead of dashboard
